### ISSUE TITLE
Do you have support for Drag and Drop?

### ISSUE THREAD

**Issue by mew22** [henryblue/TvRecyclerView#9]
Hello, 
Thanks for your implementation. 
Do you have support for Drag and Drop? In basic recyclerview, it has support by extending ItemTouchHelper.Callback, but on TV touch event doesn't really exist ..

BR,
Sebastien

**Comment by henryblue** [comment]
Thanks, 
TvRecyclerView is based on TV and does not support drag and drop. You can do this with the adapter.notifyxxxx method, but you need to handle the focus problem specifically.
I'm also thinking about how to implement drag and drop on the TV, through the remote control

**Comment by mew22** [comment]
Yes me too, that's the point I need. I will take a look at ItemTouchHelper.Callback.startDrag method but I think you are right about using adapter.notifyxxx.

BR,
Sebastien

**Comment by mew22** [comment]
I see that open source TvApp use recyclerview for their menu, I will use it for my project.
